src/Compilers/VisualBasic/Portable/Binding/BinderFactory.vb

```diff
@@ -342,6 +342,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                         Select Case methodSyntax.Kind
                             Case SyntaxKind.SubNewStatement,
                                 SyntaxKind.FunctionStatement,
+                                SyntaxKind.OperatorStatement,
```

Comment:
>SyntaxKind.OperatorStatement [](start = 32, length = 28)

I think VisitParameter should still be adjusted to not call this method with NodeUsage.ParameterDefaultValue unless position is actually within the default value.